docs/remote/devcontainerjson-reference.md

```diff
@@ -52,8 +52,8 @@ While some devcontainer.json properties apply generally, others are only used in
 | `forwardPorts` | array | An array of ports that should be forwarded from inside the container to the local machine. |
 | `postCreateCommand` | string,<br>array | A command string or list of command arguments to run **inside** the container after is created. The commands execute from the `workspaceFolder` in the container. Use `&&` in a string to execute multiple commands. For example, `"yarn install"` or `"apt-get update && apt-get install -y curl"`. The array syntax `["yarn", "install"]` will invoke the command (in this case `yarn`) directly without using a shell. <br />It fires after your source code has been mounted, so you can also run shell scripts from your source tree. For example: `bash scripts/install-dev-tools.sh`. Not set by default. |
 | `postStartCommand` | string,<br>array | A command string or list of command arguments to run when the container starts (in all cases). The parameters behave exactly like `postCreateCommand`, but the commands execute on start rather than create. Not set by default.|
-| `postAttachCommand` | string,<br>array | A command string or list of command arguments to run after VS Code has attached to a running container (in all cases). The parameters behave exactly like `postCreateCommand`, but the commands execute on attach rather than create. Not set by default. <br /><br /> ⚠️ Not yet supported in Codespaces. |
-| `initializeCommand` | string,<br>array | A command string or list of command arguments to run on the **local machine** before the container is created. This runs either when the container image is being built and when the running container is created or started. The commands execute from the `workspaceFolder` locally. For example, `"yarn install"`. The array syntax `["yarn", "install"]` will invoke the command (in this case `yarn`) directly without using a shell.<br /><br /> ⚠️ The command is run wherever the source code is located. For Codespaces this is in the cloud. |
+| `postAttachCommand` | string,<br>array | A command string or list of command arguments to run after VS Code has attached to a running container (in all cases). The parameters behave exactly like `postCreateCommand`, but the commands execute on attach rather than create. Not set by default. |
+| `initializeCommand` | string,<br>array | A command string or list of command arguments to run on the **local machine** before the container is created. This runs either when the container image is being built and when the running container is created or started. The commands execute from the `workspaceFolder` locally. The array syntax `["yarn", "install"]` will invoke the command (in this case `yarn`) directly without using a shell but **supports Windows/macOS/Linux path translation**. The string syntax (`"yarn install"`) is better for simple commnds.<br /><br /> ⚠️ The command is run wherever the source code is located. For Codespaces this is in the cloud. |
```

Comment:
I wonder if the first line `A command string or list of command arguments to run on the **local machine** before the container is created` should be rephrased to encapsulate web Codespaces? From my understanding, the final warning `For Codespaces this is in the cloud` addresses it, so I guess it isn't a big deal, but on first read, I was thinking, "What if I'm in the web? What's considered the local machine?"

Reply:
I know this line wasn't updated in this PR: `This runs either when the container image is being built and when the running container is created or started`, but for some reason it's throwing me off. 

I think the use of "either" isn't clear - what are the two cases? Either 1) when the image is being built, or 2) when the container is created and started? OR, is it 1) when the image is being built and the running container is created, or 2) the container is started? 

Maybe I'm overthinking this/it's minutiae? 😄

Reply:
Switched to referencing a "host" and clarifying later.